lib/find-view.coffee

```diff
@@ -208,8 +208,12 @@ class FindView extends View
   focusFindEditor: =>
     selectedText = atom.workspace.getActiveTextEditor()?.getSelectedText?()
     if selectedText and selectedText.indexOf('\n') < 0
+      @deactivateSelectionOption()
       selectedText = Util.escapeRegex(selectedText) if @model.getFindOptions().useRegex
       @findEditor.setText(selectedText)
+    else if selectedText and selectedText.indexOf('\n') > 0
+      @activateSelectionOption()
+      @findEditor.setText("")
```

Comment:
Here, it should not clear the Text.
This is a natural workflow for search in selection.
1. select the word to search, then ctrl-f.
2. select the paragraph, then ctrl-f again. Now search in selection is activated, and text to search is not changed.